superset/commands/base.py

```diff
@@ -26,7 +24,7 @@ class BaseCommand(ABC):
     """
 
     @abstractmethod
-    def run(self) -> Optional[Model]:
+    def run(self) -> Any:
```

Comment:
@john-bodley I'm setting base has Any, a bit wide, another option could be maintaining `Optional[Model]`